src/cmd_line/commands/yank.ts

```diff
@@ -34,7 +61,12 @@ export class YankCommand extends CommandBase {
   }
 
   async execute(vimState: VimState): Promise<void> {
-    await this.yank(vimState, vimState.cursorStartPosition, vimState.cursorStopPosition);
+    const linesToYank = this.arguments.linesToYank;
+    const startPosition = vimState.cursorStartPosition;
+    const endPosition = linesToYank
+      ? startPosition.getDown(linesToYank - 1).getLineEnd()
+      : vimState.cursorStopPosition;
+    await this.yank(vimState, startPosition, endPosition);
   }
 
   override async executeWithRange(vimState: VimState, range: LineRange): Promise<void> {
```

Comment:
Seems like `[count]` is used if it's given, even if there's a range as well

Reply:
I tested this with `:[%, $, number]yank [reg] [cnt]` and `executeWithRange(...)` is called instead of `execute(...)`, in other words, I haven't encountered this behaviour. Is there a particular command where you saw this happen?

Reply:
Sorry, I mean Vim uses `[count]` if it's given, even with a range. So with VisualLine highlighting of lines 2 and 3 here:
```
one
two
three
four
five
six
```
`:y3` should copy
```
four
five
six
```

Reply:
Added support for `[range]` with `[cnt]` argument to `:delete` and `:yank`, and added a test suite for `:yank`. The build and tests work locally, not sure why Prettier is failing in the Github Action with:
```
Amending the current commit...
You are not currently on a branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>
```
Can I manually rerun the action?

Reply:
Thanks!

Yeah, it sees that the formatting isn't right and tries to fix that for you, but I think the action is misconfigured or something. In the future, `gulp forceprettier` should fix it for you.